src/middlewares/patient.js

```diff
@@ -3,18 +3,26 @@
 
     var keyFor = {
         "Observation": "subject",
-        "MedicationPrescription": "patient"
+        "MedicationPrescription": "patient",
+        "Encounter": "patient"
```

Comment:
Good. Let's do both then - bind variants of the client to`smart.api` and `smart.patient.api` in the SMART JS Client and have the sanity check list in `fhir.js` (granted, it's something else to maintain, but makes things a bit nicer from a user perspective)